experimental/dds/tree2/src/feature-libraries/sequence-field/rebase.ts

```diff
@@ -449,11 +478,11 @@ function rebaseMark<TNodeChange>(
 function markFollowsMoves(mark: Mark<unknown>): boolean {
 	const type = mark.type;
 	switch (type) {
+		case "Insert":
 		case "Delete":
 		case "MoveOut":
+		case "Transient":
 			return true;
```

Comment:
The insert and transient cases are a little wierd because we wouldn't consider an new attach to follow a move. This code is correct because of the assumtion that, in those cases, we won't be invoking this function.
How about a comment or some asserts?

Reply:
I've added an assert.